File: Java/SpringDemo/src/main/java/io.innofang/inner_bean/Person.java
package io.innofang.inner_bean;

/**
 * Created by Inno Fang on 2018/3/30.
 */
public class Person {
    private String name;
    private String address;
    private int age;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "Student{" +
                "name='" + name + '\'' +
                ", address='" + address + '\'' +
                ", age=" + age +
                '}';
    }
}

File: Kotlin/introduction/Person.kt

/**
 * Created by Inno Fang on 2017/11/2.
 */
data class Person(val name: String, val age: Int? = null)

fun main(args: Array<String>) {
    val persons = listOf(Person("Alice"),
            Person("Bob", age = 29))

    val oldest = persons.maxBy { it.age ?: 0 }

    println("The oldest is: $oldest")
}